src/sysv-generator/sysv-generator.c

```diff
@@ -788,6 +788,10 @@ static int enumerate_sysv(const LookupPaths *lp, Hashmap *all_services) {
                         if (!fpath)
                                 return log_oom();
 
+                        log_warning("SysV service '%s' lacks a native systemd unit file. "
+                                    "Automatically generating a unit file for compatibility. "
+                                    "Please update package to include a native systemd unit file, in order to make it more safe and robust.", fpath);
```

Comment:
I'm not convinced that those warnings without any new lines will be readable. But maybe let's merge this, and adjust the format later if necessary.